app/views/directives/edit-environment-from.html

```diff
@@ -78,4 +84,44 @@
         ng-click="$ctrl.onAddRow()">{{ $ctrl.addRowLink }}</a>
     </div>
   </div>
+
+  <overlay-panel class="add-config-to-application" show-panel="$ctrl.overlayPanelVisible" show-close="true" handle-close="$ctrl.closeOverlayPanel">
+    <div class="dialog-title">
+      <h3>Value Details</h3>
+    </div>
+    <div class="modal-body">
+      <h4>{{$ctrl.overlayPaneEntryDetails.metadata.name}}
+        <small class="muted">- {{$ctrl.overlayPaneEntryDetails.kind | humanizeKind : true}}</small></h4>
+
+      <div ng-if="!($ctrl.overlayPaneEntryDetails.data | size)" class="empty-state-message text-center">
+        The {{$ctrl.overlayPaneEntryDetails.kind | humanizeKind : true}} has no properties.
+      </div>
+
+      <div ng-if="$ctrl.overlayPaneEntryDetails.data | size" class="table-responsive scroll-shadows-horizontal">
+        <table class="table table-bordered table-bordered-columns config-map-table key-value-table">
+          <tbody>
+            <tr ng-repeat="(prop, value) in $ctrl.overlayPaneEntryDetails.data">
+              <td class="key">{{prop}}</td>
+              <td class="value">
+                <truncate-long-text
+                  ng-if="$ctrl.overlayPaneEntryDetails.kind === 'ConfigMap'"
+                  content="value"
+                  limit="1024"
+                  newline-limit="20"
```

Comment:
Probably want to make these limits smaller since it's shown in a dailog.

Reply:
heh, agreed!